### PR TITLE
[bitnami/minio]: Option to use a secret.

### PR DESCRIPTION
**Pull request by ed382**
### Description of the change

In some environments, Kubernetes secrets are not allowed and credentials are mounted using some other mechanism (i.e. Vault sidecar injector). 

### Benefits

This offers the option to operate without a Kubernetes secret.

### Possible drawbacks

n/a

### Applicable issues

n/a

### Additional information

n/a

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thank you for initiating this pull request. We appreciate your effort.
There is automation in charge of automatically updating the README, CRDs and CHANGELOG, to be able to do that, it should commit to the branch from where the PR is created.

Usually, forks allow those commits by default, if there is any restriction in place you need to add @bitnami-bot as a collaborator in the [kubeshop/bitnami-charts](https://github.com/kubeshop/bitnami-charts/') repo

File: bitnami/minio/templates/standalone/deployment.yaml [hunk 1]
```diff
@@ -231,8 +231,7 @@ spec:
             - name: empty-dir
               mountPath: /.mc
               subPath: app-mc-dir
-            {{- if .Values.auth.useCredentialsFiles }}
-            - name: minio-credentials
```
Comment (alemorcuq) [inline bitnami/minio/templates/standalone/deployment.yaml:235]: I think this is a typo.

**Review by alemorcuq — CHANGES_REQUESTED** [review]

**Comment by carrodher** [comment]
Sorry for the late response, some conflicts need to be resolved before proceeding. Could you please rebase from the `main` branch? Thanks

**Review by jotamartos — DISMISSED** [review]

**Comment by jotamartos** [comment]
Changes look good to me but can you rebase your branch?

**Review by jotamartos — APPROVED** [review]
Thanks for your contribution.

**Pull request merged**